### ISSUE TITLE
Calculation of Angular Error

### ISSUE THREAD

**Issue by varunsingh3000** [yihuacheng/GazeTR#11]
Hi, the GazeTR model outputs the yaw and pitch for a given face image, but if we wanted to calculate the angular error and possibly the PoG, what's the process of doing it, the GazeTR paper doesn't provide details about it (apologies if I missed it). I'm specifically working with the Gaze360 dataset.

I did find the methods "gazeto3d" and "angular" in gtools.py. Am I supposed to input the yaw and pitch to the "gazeto3d" method and then use the output and the 3d gaze ground truth to generate the angular error directly, or is there some step I'm skipping?

Thanks in advance.

**Comment by zorazll** [comment]
Hello, may I ask if your issue has been resolved? I am using the MPIIGaze dataset, and the average error I obtained is around 0.4, like this [80] Total Num: 45000, avg: 0.41229396885316455. How is the final error of four degrees determined in this case? Thank you very much!

**Comment by varunsingh3000** [comment]
No, not really. I followed the process I mentioned above but the error was quite high around 26.51. Not sure what exactly the process to calculate is but I haven't looked into this more.

**Comment by guascy666** [comment]
> I have completed the model training and obtained the weight file. What should I do next? I want to test the code, but the leave.py file does not have the - t parameter. How can I obtain the yaw, pitch, and line of sight error，thanks

i think it is tester/leave.py but not trainer/leaver.py
